### PR TITLE
cordova-common@3.2.0 release

### PR DESCRIPTION
**Pull request by brodybits**

### CODE REVIEW

File: www/_posts/2019-06-14-common-release-3.2.0.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+categories: news
+tags: release tools
+---
+
+We are happy to announce that `cordova-common 3.2.0` was released in June 2019. This is one of the libraries used behind-the-scenes by nearly all of the Cordova tooling and provides utilities for dealing with things like `config.xml` parsing.
+
+## Release Highlights
+
+* (ios) plist document not indented with tabs ([#69](https://github.com/apache/cordova-common/pull/69))
+* Update fs-extra to v8 ([#70](https://github.com/apache/cordova-common/pull/70))
```
Comment (janpio) [inline www/_posts/2019-06-14-common-release-3.2.0.md]:
```suggestion
* Update `fs-extra` to v8 ([#70](https://github.com/apache/cordova-common/pull/70))
```

File: www/_posts/2019-06-14-common-release-3.2.0.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* Add example usage of podspec declarations ([#67](https://github.com/apache/cordova-common/pull/67))
+* implement setPreference and setPlatformPreference ([#63](https://github.com/apache/cordova-common/pull/63))
```
Comment (janpio) [inline www/_posts/2019-06-14-common-release-3.2.0.md]:
```suggestion
* implement `setPreference` and `setPlatformPreference` ([#63](https://github.com/apache/cordova-common/pull/63))
```

File: www/_posts/2019-06-14-common-release-3.2.0.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+* Catch leaked exceptions in superspawn and convert them to rejections ([#66](https://github.com/apache/cordova-common/pull/66))
```
Comment (janpio) [inline www/_posts/2019-06-14-common-release-3.2.0.md]:
```suggestion
* Catch leaked exceptions in `superspawn` and convert them to rejections ([#66](https://github.com/apache/cordova-common/pull/66))
```

File: www/_posts/2019-06-14-common-release-3.2.0.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+tags: release tools
+---
+
+We are happy to announce that `cordova-common 3.2.0` was released in June 2019. This is one of the libraries used behind-the-scenes by nearly all of the Cordova tooling and provides utilities for dealing with things like `config.xml` parsing.
+
+## Release Highlights
+
+* (ios) plist document not indented with tabs ([#69](https://github.com/apache/cordova-common/pull/69))
+* Update fs-extra to v8 ([#70](https://github.com/apache/cordova-common/pull/70))
+* Add example usage of podspec declarations ([#67](https://github.com/apache/cordova-common/pull/67))
```
Comment (janpio) [inline www/_posts/2019-06-14-common-release-3.2.0.md]:
```suggestion
* Add example usage of `podspec` declarations ([#67](https://github.com/apache/cordova-common/pull/67))
```

File: www/_posts/2019-06-14-common-release-3.2.0.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+title:  "Cordova-Common Release"
+categories: news
+tags: release tools
+---
+
+We are happy to announce that `cordova-common 3.2.0` was released in June 2019. This is one of the libraries used behind-the-scenes by nearly all of the Cordova tooling and provides utilities for dealing with things like `config.xml` parsing.
+
+## Release Highlights
+
+* (ios) plist document not indented with tabs ([#69](https://github.com/apache/cordova-common/pull/69))
```
Comment (janpio) [inline www/_posts/2019-06-14-common-release-3.2.0.md]:
```suggestion
* (ios) `plist` document not indented with tabs ([#69](https://github.com/apache/cordova-common/pull/69))
```

**Comment by brodybits** [comment]
Thanks @janpio for the suggestions. I hope this will be a squash merge.

**Pull request merged**